src/frontend/pages/blogs/BlogPage.js

```diff
@@ -0,0 +1,206 @@
+import React, { useCallback } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components';
+import UnstyledButton from '../../components/Button';
+import PreviewTable from '../../components/PreviewTable';
+import TableCell from '@material-ui/core/TableCell';
+import useBlogInfo from '../../hooks/blogs'
+import { useHistory } from 'react-router-dom';
+import closeIcon from './assets/closeIcon.svg';
+
+const Button = styled(UnstyledButton)``;
+
+const Container = styled.div`
+  margin: ${(props) => props.theme.pageMargin};
+`;
+
+const CurrentGooseHeader = styled.p`
+  font: ${({ theme }) => theme.fonts.medium24};
+`;
+
+const ButtonContainer = styled.div`
+  ${Button} {
+    margin-right: 28px;
+    box-shadow: ${({ theme }) => theme.shadows.shadow1};
+  }
+`;
+
+const TextBold = styled.p`
+  font: ${({ theme }) => theme.fonts.bold18};
+`;
+
+const SummaryBtn = styled.button`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 95px;
+  height: 36px;
+  border: 1px solid ${(props) => props.theme.colours.yellows.yellow2};
+  font: ${(props) => props.theme.fonts.bold18};
+  line-height: 23px;
+  background-color: ${(props) => props.theme.colours.yellows.yellow2};
+  color: ${(props) => props.theme.colours.blacks.black3};
+  box-sizing: border-box;
+  border-radius: 15px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colours.greys.grey2};
+    border: 1px solid ${(props) => props.theme.colours.greys.grey2};
```

Comment:
I think Jeff commented on this earlier. This is a Yellow Button. Do you know if CMS already has a pre-made Yellow button component you can important and use ?